manager/client.go

```diff
@@ -59,6 +63,20 @@ func (cm *ClientManager) RemoveClient(client Client) {
 	close(client.payloadChan)
 }
 
+func (cm *ClientManager) DeleteEvents(client *Client) {
```

Comment:
__[golint]__
* [golint] exported method ClientManager.DeleteEvents should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->